clang/test/CIR/CodeGen/openmp.cpp

```diff
@@ -0,0 +1,36 @@
+// RUN: %clang_cc1 -std=c++20 -triple x86_64-unknown-linux-gnu -clangir-disable-emit-cxx-default -fopenmp -fclangir-enable -emit-cir %s -o %t.cir
```

Comment:
You can probably remove `-clangir-disable-emit-cxx-default` from this test (at least for now).